install.zsh

```diff
@@ -8,16 +11,16 @@ NVIM_HOME="$(realpath ~)/.config/nvim"
 ROOT=$(realpath ../)
 CONFIG=$(realpath ./config)
 
-# clone NvChad along side this folder
+# Clone NvChad along side this folder
```

Comment:
Extremely important that we capitalize the first letters in the comments so they match the uninstll script format. :-)